bundles/SystemInfoBundle/public/js/startup.js

```diff
@@ -0,0 +1,99 @@
+pimcore.registerNS('pimcore.bundle.SystemInfo');
```

Comment:
```suggestion
pimcore.registerNS('pimcore.bundle.system_info.startup');
```
should be lower case, not sure if should be snake case though 
also change accordingly line 3 and 99